eth/downloader/downloader.go

```diff
@@ -369,7 +382,7 @@ func (d *Downloader) syncWithPeer(p *peer, hash common.Hash, td *big.Int) (err e
 		if d.syncInitHook != nil {
 			d.syncInitHook(origin, latest)
 		}
-		return d.spawnSync(
+		return d.spawnSync(origin+1,
```

Comment:
Actually, I'd rather leave it like this because `origin+1` is a mandatory argument, whereas all the remaining ones are variadic arguments of the same callback types. To me at least this makes for a nicer reading. If you feel strongly then I can change it, but I think the current structure highlights the conceptual differences between the two sets of parameters (mandatory, variadic) better.